src/front-end/src/helper-methods.js

```diff
@@ -2,6 +2,27 @@ import firebase from './firebase-init';
 
 const auth = firebase.auth();
 
+export function resendVerificationEmail(user) {
+  user
+    .sendEmailVerification()
+    .then(() => {
+      window.alert('A new verification email has successfully been sent!');
+      console.log('Success');
+    })
+    .catch((error) => {
+      console.log(error);
+      switch (error.code) {
+        case 'auth/too-many-requests':
+          window.alert(
+            'Please check the inbox and junk folder of your email before pressing the button again',
```

Comment:
I'd change this to "Too many requests. Please check the inbox and junk folder of your email before pressing the button again." just so the user understands why we're saying that.